tutorials.html

```diff
@@ -0,0 +1,210 @@
+          <li>PySpark Gateway</li>
+          <li>Three challenges: Adapting Spark RDD to geospatial 2D/3D</li>
+          <li>Parallel computing styles: parallelize over time/over space/variable, model, metrics, parameters</li>
+          <li>sciRDD transformation and actions</li>
+          <li>sciSpark extensions for netCDF</li>
+          <li>SciSpark front-end: scala, python, spark sql; Notebooks automatic connects to spark-shell</li>
+          <li>Apache Zeppelin, SciSpark, sciRDD</li>
+          <li>Virtual machines with SciSpark were given to attendees.</li>
+        </ol>
+        <p>Warm up -101-1: intro. To Spark: some basic examples, such as work count<br>
+        101-2 SparkSQL and DataFrames: USEFUL FOR EARTH scientific data: e.g. csv<br>
+        Using the SparkSQL package for discovery within the Storm Database<br>
+        Load csv data -&gt; clean data -&gt; create schema for Dataframe -&gt; run Spark SQL to query the dataframe with ‘typical’ RDD operations.</p>
+        <p>Parallel Statistical Rollups for a Time-Series of Grids</p>
+        <ol>
+          <li>Compute per-pixel</li>
+          <li>Statistic algorithms:</li>
+          <li>Rollup statistics by Daily, monthly, yearly</li>
+        </ol>
+        <p>Demo: Parallel Statistical Rollups for a Time-Series of Grids</p>
+        <ol>
+          <li>Read files using OpeNDAP, and split URL’s by month;</li>
+          <li>Define accumulate function -&gt; update accumulators for a set of variable grids</li>
+          <li>Define combine function to merge accumulators to Go from Monthly to seasonal to yearly to total</li>
+          <li>Define function to compute final statistics from the accumulators</li>
+          <li>Define function to write stats to netcdf file</li>
+        </ol>
+
+        <h3>SciSpark 201: Searching for MCCs</h3>
+
+        <strong>Abstract/Agenda:</strong>
+        <p>We introduce a 3 part course module on <a href="http://sciscpark.jpl.nasa.gov">SciSpark</a>, our AIST14 funded project for Highly Interactive and Scalable Climate Model Metrics and Analytics. The three part course session includes 101, 201, and 301 classes for learning how to use Spark for science.</p>
+        <p>SciSpark 201 is a 1.5 hour session in which we will use the search for Mesoscale Convective Complexes (MCCs) in Satellite Infrared data to show a real world example of how SciSpark enables real time response to both search queries and modifications to the underlying code. This task is representative of the motivation behind SciSpark - &nbsp;iterative data-reuse algorithms that share information between multiple stages.</p>
+        <ul>
+          <li>Whitehall, Kim, et al. "Exploring a graph theory based algorithm for automated identification and characterization of large mesoscale convective systems in satellite datasets." Earth Science Informatics 8.3 (2015): 663-675.</li>
```

Comment:
I just copied this text directly from the ESIP events. If you want a different reference style, please tell me what you prefer.

As far as Grab 'Em, Graph 'Em, and Tag 'Em: you have "Grab 'em, Tag 'em, Graph 'em (GTG) algorithm" on your github repo page, so please specify which I should use on the tutorials page to be consistent: https://github.com/kwhitehall/grab-tag-graph

Thanks.